tests/MyGame/Example/Ability.cs

```diff
@@ -26,6 +28,28 @@ public struct Ability : IFlatbufferObject
     builder.PutUint(Id);
     return new Offset<MyGame.Example.Ability>(builder.Offset);
   }
+  public AbilityT UnPack() {
+    var _o = new AbilityT();
+    this.UnPackTo(_o);
+    return _o;
+  }
+  public void UnPackTo(AbilityT _o) {
+    _o.Id = this.Id;
+    _o.Distance = this.Distance;
+  }
+  public static Offset<MyGame.Example.Ability> Pack(FlatBufferBuilder builder, AbilityT _o) {
+    if (_o == null) return default(Offset<MyGame.Example.Ability>);
+    return CreateAbility(
+      builder,
+      _o.Id,
+      _o.Distance);
+  }
+};
+
+public class AbilityT
+{
+  public uint Id { get; set; } = 0;
```

Comment:
@aardappel FYI this syntax is C#6 (.NET 4.6), which implies a minimal framework version. I am fine with this, but it's up to you as to whether you had aspirations for FB working for a specific version in .NET

Reply:
Please set the minimum requirement of the C# version and .NET implementation. (.NET Standard or .NET Core, Mono, etc)
I will comply with your decision.

Reply:
I am not particularly knowledgable about C# versions and what platforms support them.
I know it is important we support Unity, which is always somewhat behind.
Would be nice if we don't increase the C# version requirements for no reason beyond what we already required (which is what?).. since we're generating code, a bit of verbosity doesn't hurt, so if this can be written easily in an older style that probably would be good to do regardless.

Reply:
unfortunately, I am not Unity engineer, And I don't know Unity ecosystem too.
Looks like Unity 2019 support C# 7.3, .NET Standard 2.0. 🎉 
but according to these issues #4811 #4935, some FB users still use Unity 5(2015 version of Unity).
so maybe we should support C# 3.0, .NET Framework 3.5, VS2010...  😩

Reply:
OK, I try rewriting with the old style.